src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +610,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(String name, String flag) {
+		return flag + " " + name;
+	}
+
+	private String description(String desc, String networks) {
```

Comment:
Sorry, I overlooked the annotation when reading your response. I did test it, however I messed up my commit :-1:  The last commit should have fixed that now.